docs/reference/ilm/actions/ilm-forcemerge.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[ilm-forcemerge]]
+=== Force Merge
+
+Phases allowed: hot, warm.
+
+NOTE: Index will be be made read-only when this action is run
+(see: <<dynamic-index-settings,index.blocks.write>>)
+
+NOTE: If the `forcemerge` action is used in the `hot` phase, the `rollover` action *must* be preset.
+ILM validates this predicate and will refuse a policy with a forcemerge in the hot phase without a
+rollover action.
```

Comment:
Not a huge fan of stacking admonitions, but it may be necessary here. Should probably move them after the action description though.

Reply:
Agree we should avoid stack admonitions whenever possible.